lightning/src/ln/splicing_tests.rs

```diff
@@ -157,3 +157,61 @@ fn test_v1_splice_in() {
 		acceptor_node.node.get_our_node_id()
 	);
 }
+
+#[test]
+fn test_v1_splice_in_negative_inufficient_inputs() {
+	let chanmon_cfgs = create_chanmon_cfgs(2);
+	let node_cfgs = create_node_cfgs(2, &chanmon_cfgs);
+	let node_chanmgrs = create_node_chanmgrs(2, &node_cfgs, &[None, None]);
+	let nodes = create_network(2, &node_cfgs, &node_chanmgrs);
+
+	// Initiator and Acceptor nodes
+	let initiator_node_index = 0;
+	let acceptor_node_index = 1;
+	let initiator_node = &nodes[initiator_node_index];
+	let acceptor_node = &nodes[acceptor_node_index];
+
+	// Instantiate channel parameters where we push the maximum msats given our funding satoshis
```

Comment:
What is being pushed? In general, I'm not sure how useful comments like this are.